bcdaworker/main_test.go

```diff
@@ -2,12 +2,12 @@ package main
 
 import (
```

Comment:
Most of the changes to this file was a result of pushing up more state into `SetupSuite()` and `SetupTest()`